system/default_ad_api/test/main.test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 TIER IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import importlib.util
+import unittest
+
+from ament_index_python.packages import get_package_share_directory
+import launch
+import launch_testing.actions
+import launch_testing.markers
+import launch_testing.tools
+import pytest
+
+
+@pytest.mark.launch_test
+@launch_testing.markers.keep_alive
+def generate_test_description():
+    path = get_package_share_directory("default_ad_api") + "/launch/default_ad_api.launch.py"
+    spec = importlib.util.spec_from_file_location("temp", path)
```

Comment:
If `temp` has no meaning, is `dummy` or `dummy_module` better?

```suggestion
    spec = importlib.util.spec_from_file_location("dummy", path)
```

Reply:
This process is reading launch script (default_ad_api.launch.py). So fixed to `launch_script`.